README.md

```diff
@@ -104,7 +104,7 @@ Error Codes
 | NQA001 | "`#noqa`" must have a single space after the hash, e.g. "`# noqa`" |
 | NQA002 | "`# noqa X000`" must have a colon, e.g. "`# noqa: X000`" |
 | NQA003 | "`# noqa : X000`" must not have a space before the colon, e.g. "# noqa: X000"' |
-| NQA004 | "`# noqa:  X000`" must have at most one space before the codes, e.g. "`# noqa: X000`" |
+| NQA004 | "`# noqa:␣␣X000`" must have at most one space before the codes, e.g. "`# noqa: X000`" |
```

Comment:
Thanks for the fix, but I'm wondering if `&nbsp;&nbsp;` would be better than `␣␣` (and maybe replace all the other spaces in the example comments with `&nbsp;` for consistent spacing and behavior).

Reply:
I'm afraid that it does not work. &nbsp; in backticks are rendered as they are `&nbsp;&nbsp;` and without the backticks they are rendered as two spaces but in the render, they are tiny so it still looks like one.

Reply:
Without the backticks you're not getting a monospace font so it will be too small.

I agree with the basic issue, I'm just not thrilled about having spaces and  ␣ both being used. I think that'll be confusing. 

Doing some testing, it looks like `&nbsp;` entities are respected inside regular `<code>` blocks, so:
`"<code># noqa:&nbsp;&nbsp;X000</code>"` looks like it gets the job done, and keeps the look consistent (I wouldn't be bothered if all spaces in the examples got turned into `&nbsp;` for consistency and guaranteed even rendering).